test/WebJobs.Script.Tests/WebScriptHostManagerTests.cs

```diff
@@ -124,36 +125,6 @@ public void SecretFilesArePurgedOnStartup()
         }
 
         [Fact]
-        public void GetHttpFunctionOrNull_DecodesUriProperly()
```

Comment:
We used to allow spaces in function names, which required URL decode on function names. I cleaned that code up and removed it, so this test is no longer needed.